test/integration/features/cmd-addons.feature

```diff
@@ -3,13 +3,33 @@ Feature: Addons command and its subcommands
 
   @minishift-only
   Scenario: Default add-ons are installed but not enabled by default
-     Note: default addons were installed during previous commands automatically.
+     Note: default add-ons were installed during previous commands automatically.
```

Comment:
@LalatenduMohanty , just wondering how this word fix is different than other `addons`  in other feature files

https://github.com/minishift/minishift/blob/044cee24a3331d2bec503b1a30ff4e3af624526f/test/integration/features/addon-registry-route.feature#L3-L11

Reply:
@budhrg I am will fix it too.